src/Renci.SshNet/Common/SshData.cs

```diff
@@ -146,11 +151,13 @@ protected byte[] ReadBytes()
         /// <exception cref="ArgumentOutOfRangeException"><paramref name="length"/> is greater than the internal buffer size.</exception>
         protected byte[] ReadBytes(int length)
         {
+#pragma warning disable S125 // Sections of code should not be commented out
             /*
              * Note that this also prevents allocating non-relevant lengths, such as if length is greater than _data.Count but less than int.MaxValue.
              * For the nerds, the condition translates to: if (length > data.Count && length < int.MaxValue)
              * Which probably would cause all sorts of exception, most notably OutOfMemoryException.
              */
+#pragma warning restore S125 // Sections of code should not be commented out
```

Comment:
The comment seems out of date

```suggestion
```

Reply:
I've remove the comment and added unit tests for this method.